patches/react-native-scroll-bottom-sheet+0.6.1.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/node_modules/react-native-scroll-bottom-sheet/src/index.tsx b/node_modules/react-native-scroll-bottom-sheet/src/index.tsx
```

Comment:
I'm not sure it's the case, but maybe we should create a fork

Reply:
I've opened a PR at the library because it's a bug fix.
I think we can maintain the patch while it's not merged.
https://github.com/rgommezz/react-native-scroll-bottom-sheet/pull/10